### PR TITLE
Reflectometry (polref) documentation update

### LINKED ISSUE #17897: Update documentation of Reflectometry (polref) interface
This issue is to update documentation according to changes in:
- [ ] https://github.com/mantidproject/mantid/issues/16983
- [ ] https://github.com/mantidproject/mantid/issues/17816

### PR DESCRIPTION
**Pull request by PranavBahuguna**
To accompany the changes made in #16983 and #17816 to the reflectometry (polref) interface, the documentation has been updated accordingly.

**To test:**

Ensure the new documentation accurately describes the new functionality of the updated interface.

Fixes #17897.

Blocked by #17816.

*Release notes have been updated.*

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [x] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [x] Has the relevant documentation been added/updated?
- [x] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?

- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/interfaces/ISIS_Reflectometry.rst [hunk 1]
```diff
@@ -388,22 +387,93 @@ Hovering over the highlighted run with your cursor will allow you to see why the
...
+| List Of Logged Parameters     | The right listbox starts out empty, but will fill    |
+|                               | with parameter names when a workspace in the left    |
+|                               | listbox is double clicked. This listbox supports     |
+|                               | multi-select in order to allow for the save output   |
+|                               | to contain multiple parameter notes.                 |
++-------------------------------+------------------------------------------------------+
+| Spectra list                  | Spectra list entry allows you to specify what        |
+|                               | spectra you want in the output file. If this is      |
+|                               | blank, all spectra are included.                     |
++-------------------------------+------------------------------------------------------+
```
Comment (raquelalvarezbanos) [inline docs/source/interfaces/ISIS_Reflectometry.rst]: Should this be removed now that we have removed `Spectra list`?

File: docs/source/interfaces/ISIS_Reflectometry.rst [hunk 2]
```diff
@@ -388,22 +387,93 @@ Hovering over the highlighted run with your cursor will allow you to see why the
...
+| File format                   | This dialog can save to ANSTO, Ill cosmos, 3-column, |
```
Comment (raquelalvarezbanos) [inline docs/source/interfaces/ISIS_Reflectometry.rst]: Very minor detail: `Ill` -> `ILL`

File: docs/source/interfaces/ISIS_Reflectometry.rst [hunk 3]
```diff
@@ -388,22 +387,93 @@ Hovering over the highlighted run with your cursor will allow you to see why the
...
+|                               | and a customisable format. It doesn't save from      |
+|                               | the main interface's table, but from workspaces      |
+|                               | loaded into mantid. All algorithms are also          |
+|                               | available as save algorithms from mantid itself.     |
++-------------------------------+------------------------------------------------------+
+| Custom Format Options         | When saving in 'Custom' this section allows you      |
+|                               | to specify if you want a Title and/or Q Resolution   |
+|                               | headings as well as specifying the delimiter.        |
```
Comment (raquelalvarezbanos) [inline docs/source/interfaces/ISIS_Reflectometry.rst]: Is `Q Resolution` added to the heading? I thought it was added as an extra column...

File: docs/source/interfaces/ISIS_Reflectometry.rst (hunk 1 above)
Comment (raquelalvarezbanos) [inline docs/source/interfaces/ISIS_Reflectometry.rst]: Also, figure `docs/source/images/ISISReflectometryPolref_save_tab.png` will need to be updated too

File: docs/source/algorithms/StartLiveData-v1.rst [hunk 4]
```diff
@@ -1,240 +0,0 @@
...
-.. testoutput:: exStartLiveDataHisto
-   :options: +ELLIPSIS, +NORMALIZE_WHITESPACE
-
-    The workspace contains ... periods
-    Each period   contains ... spectra
-
-
-.. categories::
-
-.. sourcelink::
```
Comment (raquelalvarezbanos) [inline docs/source/algorithms/StartLiveData-v1.rst]: Why was this file removed?

File: docs/source/interfaces/ISIS_Reflectometry.rst [hunk 5]
```diff
@@ -53,17 +53,17 @@ dialog will open. Select ``INTER_NR_test2.tbl`` as the file, and enter ``MyTable
 as the output workspace.
 
 A table workspace called ``MyTable`` should now exist in the ADS (:ref:`Analysis Data Service <Analysis Data Service>`).
-To open the table workspace go to **Reflectometry -> Open Table -> MyTable**.
-The processing table (shown below) should now contain four rows (13460, 13462, 13469, 13470).
+In addition the table workspace should be opened as well and  The processing table
```
Comment (raquelalvarezbanos) [inline docs/source/interfaces/ISIS_Reflectometry.rst]: Thanks for correcting this. Minor detail: `... and  The processing ...` -> `... and the processing ...`

File: docs/source/interfaces/ISIS_Reflectometry.rst [hunk 6]
```diff
@@ -53,17 +53,17 @@ dialog will open. Select ``INTER_NR_test2.tbl`` as the file, and enter ``MyTable
...
-.. figure:: /images/ISISReflectometryPolref_INTER_table.JPG
+.. figure:: /images/ISISReflectometryPolref_INTER_table.PNG
   :align: center
 
 Let's process the first group, which consists of the first two rows of the
 table (13460 and 13462). The simplest way to do this is simply to select the
 two rows we want to process, and then click on **Process**. Note that for the reduction
 to be successful, at least the bin size must be specified to :ref:`Stitch1DMany <algm-Stitch1DMany>`,
-as shown above.
+in the options tab (e.g. Params="-0.04").
```
Comment (raquelalvarezbanos) [inline docs/source/interfaces/ISIS_Reflectometry.rst]: :+1:

File: docs/source/interfaces/ISIS_Reflectometry.rst [hunk 7]
```diff
@@ -363,7 +362,7 @@ Measure based search transfer uses the log-values within nexus files from the ex
 
 - Any runs with the measurement_id log, will be
```
Comment (raquelalvarezbanos) [inline docs/source/interfaces/ISIS_Reflectometry.rst]: I know you haven't changed this, but could we replace "measurement_id" with "``measurement_id``" so that it is highlighted the same way as in the next paragraph?

File: docs/source/interfaces/ISIS_Reflectometry.rst [hunk 8]
```diff
@@ -363,7 +362,7 @@ Measure based search transfer uses the log-values within nexus files from the ex
...
   placed into the same group.
-- Any runs with the ``same measurement_id`` and the same ``measurement_subid`` logs, will be merged into a single row, with all the runs listed in the **Run(s)** column in the format, ``123+124+125``. 
+- Any runs with the ``same measurement_id`` and the same ``measurement_subid`` logs, will be merged into a single row, with all the runs listed in the **Run(s)** column in the format, ``123+124+125``.
```
Comment (raquelalvarezbanos) [inline docs/source/interfaces/ISIS_Reflectometry.rst]: Again, nothing you've modified, but could we replace "``same measurement_id``" with "same ``measurement_id``"?

**Comment by raquelalvarezbanos** [comment]
Sorry @PranavBahuguna I didn't realise this was "in progress" and started reviewing...

File: docs/source/interfaces/ISIS_Reflectometry.rst (hunk 3 above)
Comment (PranavBahuguna) [inline docs/source/interfaces/ISIS_Reflectometry.rst]: I copied a lot of these from the old ISIS Reflectometry docs. I suppose even they got some things wrong.

File: docs/source/algorithms/StartLiveData-v1.rst (hunk 4 above)
Comment (PranavBahuguna) [inline docs/source/algorithms/StartLiveData-v1.rst]: Unintentional, will replace.

File: docs/source/interfaces/ISIS_Reflectometry.rst (hunk 3 above)
Comment (raquelalvarezbanos) [inline docs/source/interfaces/ISIS_Reflectometry.rst]: OK, just check this before making any change, because I am not 100% sure I am correct.

File: docs/source/algorithms/StartLiveData-v1.rst (hunk 4 above)
Comment (raquelalvarezbanos) [inline docs/source/algorithms/StartLiveData-v1.rst]: :+1:

**Comment by raquelalvarezbanos** [comment]
There's a conflict with master, and OSX is failing for some reason

**Comment by raquelalvarezbanos** [comment]
Looks good :shipit:
